.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
.

Reply:
This is like the third time that I upload unwanted files lol